acestep/ui/gradio/interfaces/theme.py

```
@@ -0,0 +1,127 @@
+        self,
+        *,
+        primary_hue: colors.Color | str = colors.slate,
+        secondary_hue: colors.Color | str = colors.indigo,
+        neutral_hue: colors.Color | str = colors.gray,
+        spacing_size: sizes.Size | str = sizes.spacing_md,
+        radius_size: sizes.Size | str = sizes.radius_md,
+        text_size: sizes.Size | str = sizes.text_md,
+        font: fonts.Font
+        | str
+        | Iterable[fonts.Font | str] = (
+            fonts.LocalFont("Montserrat"),
```

Ruff (0.15.2)

[warning] Do not perform function call `fonts.LocalFont` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)